From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Wed, 23 Jan 2013 21:15:21 +0100
Subject: Proper install directory

---
 CMakeLists.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -316,8 +316,8 @@ IF (MSVC)
 endif()
 
 # DEFINITION : du dossier d'installation
-set(Install_Dir ${PROJECT_SOURCE_DIR}/bin)
-set(Install_Dir_lib ${PROJECT_SOURCE_DIR}/lib)
+set(Install_Dir /usr/bin)
+#set(Install_Dir_lib ${PROJECT_SOURCE_DIR}/lib)
 
 # DEFINITION : du dossier des sources d'Elise
 set(ELISE_DIR src)
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -182,7 +182,7 @@ if(${CUDA_ENABLED})
 
 else()
 	 add_library( elise ${Elise_Src_Files} ${IncFiles} )
-	 INSTALL(TARGETS elise ARCHIVE DESTINATION ${Install_Dir_lib} )
+     #INSTALL(TARGETS elise ARCHIVE DESTINATION ${Install_Dir_lib} )
 endif()
 
 # OPTION BUILD : En tetes precompilees pour gcc
