From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Thu, 24 Jan 2013 11:17:34 +0100
Subject: Hardcoded paths. FHS complient

--- a/src/util/cGPAO.cpp
+++ b/src/util/cGPAO.cpp
@@ -53,7 +53,7 @@ void cEl_GPAO::DoComInParal(const std::l
        aNbProc = NbProcSys();
 
     if (FileMk=="") 
-       FileMk = MMDir() + "MkStdMM";
+       FileMk = std::string("MkStdMM");
 
     cEl_GPAO aGPAO;
     int aK=0;
@@ -103,7 +103,7 @@ void MkFMapCmd
        aNbProc = NbProcSys();
 
     if (FileMk=="") 
-       FileMk = MMDir() + "MkStdMM";
+       FileMk =  std::string("MkStdMM");
 
 
     cEl_GPAO aGPAO;
