--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -150,12 +150,12 @@ ENDIF()
 
 # VERIFICATION si ImageMagick est installé
 
-find_package(ImageMagick REQUIRED)
+#find_package(ImageMagick REQUIRED)
 
-if(NOT ImageMagick_FOUND)
-	MESSAGE("ATTENTION!!! Vous devez installer Image Magick.")
-	MESSAGE("http://www.imagemagick.org/script/index.php")
-endif(NOT ImageMagick_FOUND)
+#if(NOT ImageMagick_FOUND)
+#	MESSAGE("ATTENTION!!! Vous devez installer Image Magick.")
+#	MESSAGE("http://www.imagemagick.org/script/index.php")
+#endif(NOT ImageMagick_FOUND)
 
 # VERIFICATION si exiftool est installé
 
