--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -323,7 +323,7 @@ set(Install_Dir /usr/bin)
 set(ELISE_DIR src)
 
 # CONSTRUCTION : Librairie ANN
-include(${PROJECT_SOURCE_DIR}/CodeExterne/ANN/CMakeLists.txt)
+# include(${PROJECT_SOURCE_DIR}/CodeExterne/ANN/CMakeLists.txt)
 # CONSTRUCTION : Librairie ELISE
 set( code_genere_dir ${PROJECT_SOURCE_DIR}/CodeGenere )
 include(${code_genere_dir}/Sources.cmake)
--- a/src/uti_image/Ann/AnnSearcher.h
+++ b/src/uti_image/Ann/AnnSearcher.h
@@ -11,7 +11,7 @@
 #include <fstream>
 
 // this can possibly be removed if ANN is not installed
-#include "../../../CodeExterne/ANN/include/ANN/ANN.h"
+#include <ANN/ANN.h>
 
 #define SIFT_POINT_UNDEFINED_INDEX -1
 
--- a/src/CBinaires/CMakeLists.txt
+++ b/src/CBinaires/CMakeLists.txt
@@ -1,6 +1,6 @@
 add_executable(mm3d mm3d.cpp)
 
-target_link_libraries(mm3d elise ${X11_LIBRARIES} ${WINSOCK2_LIBRARY} ${DYN_LIB_LIBRARY} ANN)
+target_link_libraries(mm3d elise ${X11_LIBRARIES} ${WINSOCK2_LIBRARY} ${DYN_LIB_LIBRARY} -lann)
 
 if(${CUDA_ENABLED})
 	 target_link_libraries(mm3d ${libStatGpGpuTools} ${libStatGpGpuInterfMicMac} ${libStatGpGpuOpt})
--- a/src/uti_phgrm/CPP_Pastis.cpp
+++ b/src/uti_phgrm/CPP_Pastis.cpp
@@ -43,7 +43,7 @@ using namespace NS_ParamChantierPhotogra
 
 #if ELISE_unix
 	const std::string TheStrSiftPP = "siftpp_tgi.LINUX";
-	const std::string TheStrAnnPP  = "ann_mec_filtre.LINUX";
+	const std::string TheStrAnnPP  = "mm3d:Ann";
 #elif ELISE_MacOs
 	const std::string TheStrSiftPP = "siftpp_tgi.OSX";
 	const std::string TheStrAnnPP  = "ann_samplekey200filtre.OSX";
