From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Mon, 13 May 2013 00:00:00 +0100
Subject: Remove non needed sotware

--- a/src/CBinaires/mm3d.cpp
+++ b/src/CBinaires/mm3d.cpp
@@ -170,9 +170,10 @@ const std::vector<cMMCom> & getAvailable
        aRes.push_back(cMMCom("Campari",Campari_main," Interface to Apero, for compensation of heterogenous measures",cArgLogCom(2)));
        aRes.push_back(cMMCom("ChgSysCo",ChgSysCo_main," Chang coordinate system of orientation",cArgLogCom(2)));
        aRes.push_back(cMMCom("CmpCalib",CmpCalib_main," Do some stuff"));
-       aRes.push_back(cMMCom("cod",cod_main," Do some stuff"));
-       aRes.push_back(cMMCom("vic",vicod_main," Do some stuff"));
-       aRes.push_back(cMMCom("genmail",genmail_main," Do some stuff"));
+
+//       aRes.push_back(cMMCom("cod",cod_main," Do some stuff"));
+//       aRes.push_back(cMMCom("vic",vicod_main," Do some stuff"));
+//       aRes.push_back(cMMCom("genmail",genmail_main," Do some stuff"));
        aRes.push_back(cMMCom("CreateEpip",CreateEpip_main," Tool create epipolar images"));
        aRes.push_back(cMMCom("Dequant",Dequant_main," Tool for dequantifying an image"));
        aRes.push_back(cMMCom("Devlop",Devlop_main," Do some stuff"));
@@ -186,7 +187,7 @@ const std::vector<cMMCom> & getAvailable
        aRes.push_back(cMMCom("GCPConvert",GCP_Txt2Xml_main," Convert GCP from Txt 2 XML",cArgLogCom(3)));
        aRes.push_back(cMMCom("OriConvert",Ori_Txt2Xml_main," Convert Orientation from Txt 2 XML",cArgLogCom(3)));
 
-       aRes.push_back(cMMCom("GenXML2Cpp",GenXML2Cpp_main," Do some stuff"));
+//       aRes.push_back(cMMCom("GenXML2Cpp",GenXML2Cpp_main," Do some stuff"));
        aRes.push_back(cMMCom("GrShade",GrShade_main," Compute shading from depth image"));
        aRes.push_back(cMMCom("StackFlatField",EstimFlatField_main,"Basic Flat Field estimation by image stacking"));
        aRes.push_back(cMMCom("Impaint",Impainting_main,"Basic Impainting"));
@@ -252,7 +253,7 @@ const std::vector<cMMCom> & getAvailable
        aRes.push_back(cMMCom("ImMire",GenMire_main," For generation of some synthetic calibration image"));
        aRes.push_back(cMMCom("Undist",Undist_main," Tool for removing images distortion"));
 
-       aRes.push_back(cMMCom("CheckDependencies",CheckDependencies_main," check dependencies to third-party tools"));
+//       aRes.push_back(cMMCom("CheckDependencies",CheckDependencies_main," check dependencies to third-party tools"));
        aRes.push_back(cMMCom("VV",VideoVisage_main," A very simplified tool for 3D model of visage out of video, just for fun"));
 
        aRes.push_back(cMMCom("XYZ2Im",XYZ2Im_main," tool to transform a 3D point (text file) to their 2D proj in cam or cloud"));
--- a/src/uti_files/Sources.cmake
+++ b/src/uti_files/Sources.cmake
@@ -2,7 +2,7 @@ set(uti_files_Src_Files
     ${UTI_FILES_DIR}/CPP_MapCmd.cpp
     ${UTI_FILES_DIR}/CPP_BatchFDC.cpp
     ${UTI_FILES_DIR}/CPP_MyRename.cpp
-    ${UTI_FILES_DIR}/CPP_cod.cpp
+    #    ${UTI_FILES_DIR}/CPP_cod.cpp
     ${UTI_FILES_DIR}/CPP_GGP_txt2Xml.cpp
     ${UTI_FILES_DIR}/CPP_Ori_txt2Xml.cpp
     ${UTI_FILES_DIR}/CPP_TestKeys.cpp
