--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,7 +29,7 @@ OPTION(WITH_HEADER_PRECOMP "En-tetes pre
 OPTION(BUILD_ONLY_ELISE "Projet Elise uniquement" OFF )
 
 # construire l'interface graphique
-OPTION(WITH_ETALONPOLY  "build etalonnage polygone" ON)
+OPTION(WITH_ETALONPOLY  "build etalonnage polygone" OFF)
 
 # Utiliser Cuda
 OPTION(CUDA_ENABLED "Utilisation de cuda" OFF )
--- a/src/CBinaires/CMakeLists.txt
+++ b/src/CBinaires/CMakeLists.txt
@@ -55,8 +55,8 @@ if (NOT ${BUILD_ONLY_ELISE})
 	add_executable(Dequant  Dequant.c)
 	INSTALL(TARGETS Dequant RUNTIME DESTINATION ${Install_Dir})
 
-	add_executable(vic  vic.c)
-	INSTALL(TARGETS vic RUNTIME DESTINATION ${Install_Dir})
+    #	add_executable(vic  vic.c)
+    #INSTALL(TARGETS vic RUNTIME DESTINATION ${Install_Dir})
 
 	add_executable(Devlop  Devlop.c)
 	INSTALL(TARGETS Devlop RUNTIME DESTINATION ${Install_Dir})
--- a/src/CBinaires/mm3d.cpp
+++ b/src/CBinaires/mm3d.cpp
@@ -168,9 +168,10 @@ const std::vector<cMMCom> & getAvailable
        aRes.push_back(cMMCom("Campari",Campari_main," Interface to Apero, for compensation of heterogenous measures",cArgLogCom(2)));
        aRes.push_back(cMMCom("ChgSysCo",ChgSysCo_main," Chang coordinate system of orientation",cArgLogCom(2)));
        aRes.push_back(cMMCom("CmpCalib",CmpCalib_main," Do some stuff"));
-       aRes.push_back(cMMCom("cod",cod_main," Do some stuff"));
-       aRes.push_back(cMMCom("vic",vicod_main," Do some stuff"));
-       aRes.push_back(cMMCom("genmail",genmail_main," Do some stuff"));
+
+//       aRes.push_back(cMMCom("cod",cod_main," Do some stuff"));
+//       aRes.push_back(cMMCom("vic",vicod_main," Do some stuff"));
+//       aRes.push_back(cMMCom("genmail",genmail_main," Do some stuff"));
        aRes.push_back(cMMCom("CreateEpip",CreateEpip_main," Tool create epipolar images"));
        aRes.push_back(cMMCom("Dequant",Dequant_main," Tool for dequantifying an image"));
        aRes.push_back(cMMCom("Devlop",Devlop_main," Do some stuff"));
--- a/src/uti_files/Sources.cmake
+++ b/src/uti_files/Sources.cmake
@@ -2,7 +2,7 @@ set(uti_files_Src_Files
     ${UTI_FILES_DIR}/CPP_MapCmd.cpp
     ${UTI_FILES_DIR}/CPP_BatchFDC.cpp
     ${UTI_FILES_DIR}/CPP_MyRename.cpp
-    ${UTI_FILES_DIR}/CPP_cod.cpp
+    #    ${UTI_FILES_DIR}/CPP_cod.cpp
     ${UTI_FILES_DIR}/CPP_GGP_txt2Xml.cpp
     ${UTI_FILES_DIR}/CPP_Ori_txt2Xml.cpp
     ${UTI_FILES_DIR}/CPP_TestKeys.cpp
